server/adapters/google_cloud_storage.go

```diff
@@ -32,7 +32,7 @@ type GoogleConfig struct {
 	credentials option.ClientOption
 }
 
-func (gc *GoogleConfig) Validate(streamMode bool) error {
+func (gc *GoogleConfig) Validate(strict, streamMode bool) error {
```

Comment:
Do we really need strict parameter?
`Dataset` can be empty (it has default value 'default')
`Project` is validated on BigQuery storage creation.

Reply:
`handlers.testDestinationConnection` needs such `strict` argument. In another case, a connection will be failed.
It is needed to provide default values into adapters (for example `Project` or `Dataset`) but it is provided in the storage layer and we create duplication in this case.
Another thing that some of the values like `schema` or `database` is related to database objects which are created during preparing storage to send events. But create schemas or databases for testing connection - is a strange thing because we need to remove them after tests or in another case tests can make a lot of created schemas and databases.
So I vote that the adapter layer should be without default values, and it should be passed explicitly.
Default values can be setup in config layers or in the user layer on JS side for make connection testing.

Reply:
Reverted